### PR TITLE
Improves ElectionModule event emission

### LINKED ISSUE #934: Review EM events to include epoch index data

### PR DESCRIPTION
**Pull request by ajsantander**
Fix #934

### CODE REVIEW

**Review by andytcf — APPROVED** [review]
LFG!!!

File: packages/core-modules/contracts/submodules/election/ElectionBase.sol [hunk 1]
```diff
@@ -49,9 +49,9 @@ contract ElectionBase is ElectionStorage, InitializableMixin {
     event EpochStarted(uint epochIndex);
     event CouncilTokenCreated(address proxy, address implementation);
     event CouncilTokenUpgraded(address newImplementation);
-    event CouncilMemberAdded(address member);
-    event CouncilMemberRemoved(address member);
-    event CouncilMembersDismissed(address[] members);
+    event CouncilMemberAdded(address indexed member, uint indexed epochIndex);
+    event CouncilMemberRemoved(address indexed member);
```
Comment (andytcf) [inline packages/core-modules/contracts/submodules/election/ElectionBase.sol:53]: @ajsantander you forgot it here ser

**Review by mjlescano — APPROVED** [review]

File: packages/core-modules/contracts/submodules/election/ElectionBase.sol (hunk 1 above)
Comment (ajsantander) [reply packages/core-modules/contracts/submodules/election/ElectionBase.sol:52]: I suppose you mean that this event needs to also track `epochIndex`. On it.

Comment (ajsantander) [reply packages/core-modules/contracts/submodules/election/ElectionBase.sol]: Done!

**Pull request merged**